crates/hotshot/src/traits/election/randomized_committee.rs

```diff
@@ -0,0 +1,170 @@
+// Copyright (c) 2021-2024 Espresso Systems (espressosys.com)
+// This file is part of the HotShot repository.
+
+// You should have received a copy of the MIT License
+// along with the HotShot repository. If not, see <https://mit-license.org/>.
+
+use std::{cmp::max, collections::BTreeMap, num::NonZeroU64};
+
+use ethereum_types::U256;
+use hotshot_types::{
+    traits::{
+        election::Membership,
+        network::Topic,
+        node_implementation::NodeType,
+        signature_key::{SignatureKey, StakeTableEntryType},
+    },
+    PeerConfig,
+};
+use rand::{rngs::StdRng, Rng};
+
+#[derive(Clone, Debug, Eq, PartialEq, Hash)]
+
+/// The static committee election
+pub struct GeneralStaticCommittee<T: NodeType> {
```

Comment:
Why not just name this GeneralRandomCommittee? or similar so we don't have 2 structs named the exact same thing, I could see this causing some weird headache in the future.  Unless this is some common code style thing I wasn't aware of

Reply:
I was just following the style from before, but yeah I agree -- renamed!